root/static/styles/layout.less

```diff
@@ -126,8 +125,183 @@ pre code {
 /*
 ** Header
 */
-#header {
-    width: 100%;
+.header {
+    @menu-item-text-padding: 8px;
+    @logo-width: 166px;
+    @logo-height: 26px;
+
+    height: 62px;
+    margin-top: 10px;
+    margin-left: 16px;
+    margin-right: 16px;
+    clear: both;
+
+    img.logo {
+        width: @logo-width;
+        height: @logo-height;
+        margin-top: 16px;
+    }
+
+    .right {
+        width: ~"calc(100% - @{logo-width} - 26px)";
```

Comment:
Probably not available in all our supported browsers. :/ Looks pretty messed up in Opera 12.